dispatch/static/manager/src/js/components/inputs/DateTimeInput.js

```diff
@@ -13,25 +13,26 @@ export default function DateTimeInput(props) {
     <DateTimePicker
       value={props.value || new Date()}
       onChange={props.onChange}
-      timePickerProps={{ // so that state updates when either
-        // date or time are changed
+      timePickerProps={{
         onChange: props.onChange
       }} />
   )
 
   return (
-    <div>
+    <div className='c-input--datetime'>
       <Popover
         content={picker}
         popoverClassName='pt-popover-content-sizing'
         position={Position.RIGHT}>
```

Comment:
Super nit-picky, but can this open underneath instead of to the right? Just to be consistent with other dropdowns, and so that this would still work if it was used in the sidebar